resources/kcp/charts/keda/README.md

```diff
@@ -0,0 +1,147 @@
+| `image.metricsApiServer.tag`                               | Image tag of KEDA Metrics API Server. Optional, given app version of Helm chart is used by default | `` |
+| `crds.install`                               | Defines whether the KEDA CRDs have to be installed or not. | `true`                 |
+| `watchNamespace`                                           | Defines Kubernetes namespaces to watch to scale their workloads. Default watches all namespaces | `` |
+| `operator.name`                                            | Name of the KEDA operator | `keda-operator` |
+| `operator.replicaCount`                                      | Capability to configure the number of replicas for KEDA operator.<br /><br />While you can run more replicas of our operator, only one operator instance will be the leader and serving traffic.<br /><br />You can run multiple replicas, but they will not improve the performance of KEDA, it could only reduce downtime during a failover.| `1` |
+| `metricsServer.dnsPolicy`                                  | Defined the DNS policy for the metric server | `ClusterFirst`
+| `metricsServer.useHostNetwork`                             | Enable metric server to use host network  | `false`
+| `imagePullSecrets`                                         | Name of secret to use to pull images to use to pull Docker images | `[]` |
+| `additionalLabels`                                         | Additional labels to apply to KEDA workloads | `{}` |
+| `podAnnotations.keda`                                      | Pod annotations for KEDA operator | `{}` |
+| `podAnnotations.metricsAdapter`                            | Pod annotations for KEDA Metrics Adapter | `{}` |
+| `podLabels.keda`                                           | Pod labels for KEDA operator | `{}` |
+| `podLabels.metricsAdapter`                                 | Pod labels for KEDA Metrics Adapter | `{}` |
+| `podDisruptionBudget`                                      | Capability to configure [Pod Disruption Budget](https://kubernetes.io/docs/tasks/run-application/configure-pdb/)       | `{}` |
+| `rbac.create`                                              | Specifies whether RBAC should be used | `true`                                        |
+| `serviceAccount.create`                                    | Specifies whether a service account should be created       | `true`                                        |
+| `serviceAccount.name`                                      | The name of the service account to use. If not set and create is true, a name is generated.      | `keda-operator` |
+| `serviceAccount.annotations`                               | Annotations to add to the service account | `{}` |
+| `podIdentity.activeDirectory.identity`                     | Identity in Azure Active Directory to use for Azure pod identity | `` |

```

Comment:
```suggestion
| `podIdentity.activeDirectory.identity`                     | Identity in Azure Active Directory to use for Azure Pod identity | `` |

```

Reply:
See first comment. This was resolved after our Slack discussion.